app/routes.js

```diff
@@ -9,7 +9,7 @@ import ErrorBoundary from 'global/ErrorBoundary'
 import CookieNotice from 'global/CookieNotice'
 
 import LandingRedirect from './components/pages/LandingRedirect'
-import LoginPage from '../packages/component-login/client'
+import LoginPage from '../packages/component-login/client/LoginPage'
```

Comment:
What's the purpose of the file rename? Shouldn't this file live within the `/pages` client directory and be exported in the `index.js`? The imports on line 14 - 18 don't match how we originally did this for the Dashboard page (see line 5 above)

Reply:
I've just seen that the bellow file imports haven't been turned into packages yet. I think we should still be moving this LoginPage to a `/pages` sub-directory